main_test.go

```diff
@@ -380,7 +461,7 @@ func TestAccountHandlers(t *testing.T) {
 		{
 			name:     "details unknown address",
 			method:   http.MethodGet,
-			url:      "/0f7025fa05b578e3",
+			url:      "/e34ea67a850e1585",
```

Comment:
Address needs to be a valid address, and since there are additional accounts created with the new test cases the previous address was actually created and made the "details unknown address" case fail.